exercises/gigasecond/gigasecond.d

```diff
@@ -0,0 +1,7 @@
+module gigasecond;
+
+import std.datetime;
+
+DateTime gsAnniversary(DateTime start) {
+    return start;
```

Comment:
Should this have a `// your code here` in it to indicate more clearly that it's a stub file?